src/SFA.DAS.AssessorService.Application.Api/Validators/EpaOrganisationValidator.cs

```diff
@@ -3,29 +3,38 @@
 using Microsoft.Extensions.Logging;
 using SFA.DAS.AssessorService.Application.Exceptions;
 using SFA.DAS.AssessorService.Application.Interfaces;
+using SFA.DAS.AssessorService.ExternalApis.AssessmentOrgs;
 
 namespace SFA.DAS.AssessorService.Application.Api.Validators
 {
     public class EpaOrganisationValidator: IEpaOrganisationValidator
     {
         private readonly IRegisterQueryRepository _registerRepository;
 
-        public string ErrorMessageNoOrganisationId { get; } = "There is no organisation Id; ";
-        public string ErrorMessageOrganisationIdTooLong { get; } = "The length of the organisation Id is too long; ";
-        public string ErrorMessageOrganisationNameEmpty { get; } = "There is no organisation name; ";
-        public string ErrorMessageOrganisationIdAlreadyUsed { get; } = "There is already an entry for this organisation Id; ";
-        public string ErrorMessageUkprnAlreadyUsed { get; } = "There is already an organisation with this ukprn; ";
-        public string ErrorMessageOrganisationTypeIsInvalid { get; } = "There is no organisation type with this Id; ";
-        public string ErrorMessageAnotherOrganisationUsingTheUkprn { get; } = "The ukprn entered is already used by another organisation; ";
-        public string ErrorMessageUkprnIsInvalid { get; } = "The ukprn is not the correct format or length; ";
-        public string ErrorMessageOrganisationNotFound { get; } = "There is no organisation for the this organisation Id; ";
+        private string ErrorMessageNoOrganisationId { get; } = "There is no organisation Id; ";
+        private string ErrorMessageOrganisationIdTooLong { get; } = "The length of the organisation Id is too long; ";
```

Comment:
Not sure if we need to use localisation for string - in the web app we have doe this and also in the Api Controller to date.